### PR TITLE
Add link to Jupyter Notebooks in README.rst

### LINKED ISSUE #3182: Colab link for notebooks
https://colab.research.google.com/github/quantumlib/Cirq 

Brings up the notebooks in the project. It is kind of cool, but it requires a Google account to run them.
What do we think about adding it in the README.rst as a link to tutorials? 
(I also asked the same question in OpenFermion https://github.com/quantumlib/OpenFermion/issues/628 and we could do something similar in ReCirq as well @mpharrigan).

### PR DESCRIPTION
**Pull request by unaiic**
This change adds a link to tutorials (Jupyter Notebooks) in the README.rst.

Related to #3182.

### CODE REVIEW

**Comment by rmlarose** [comment]
Hey @unaiic I think the notebook installs have been accounted for (see the `try: import cirq` cells in each notebook) and #3182 should have been closed (@balopat)?

**Comment by unaiic** [comment]
Okay @rmlarose. Should I delete the PR, or is the link in the README.rst still useful?

**Comment by rmlarose** [comment]
> is the link in the README.rst still useful?

Definitely could be! I'll let others weigh in on this.

**Comment by balopat** [comment]
Hey @unaiic - sorry for keeping that issue lying around - I think that given that the new site has nicely integrated colab links, this is not really needed anymore that much. However, I still think that that colab link is a cool shortcut to _all_ of the notebooks available in the Cirq repo. So I'm okay to merge this.

**Comment by unaiic** [comment]
@balopat No problem. I also think it would be an interesting feature to have.

File: README.rst [hunk 1]
```diff
@@ -32,6 +32,8 @@ Documentation for the latest **stable** version of cirq (what you get if you ``p
 - `Documentation <https://quantumai.google/cirq>`_
 - `Tutorials <https://quantumai.google/cirq/tutorials>`_
 
+Run the interactive Jupyter Notebooks in `Colab <https://colab.research.google.com/github/quantumlib/Cirq>`_.
```
Comment (balopat) [inline README.rst:35]:
```suggestion
For a comprehensive list all of the interactive Jupyter Notebooks in our repo (including the ones not yet published to the site) open our repo in `Colab <https://colab.research.google.com/github/quantumlib/Cirq>`_.
```

Comment (balopat) [reply README.rst:35]: I would reword it this way, so that it's clear what users are clicking on. Can you update it please? and then I'll merge it.

Comment (unaiic) [reply README.rst]: Done.

**Review by balopat — APPROVED** [review]
